pkg/sql/sem/builtins/builtins.go

```diff
@@ -4105,7 +4106,7 @@ value if you rely on the HLC for accuracy.`,
 		}),
 
 	"crdb_internal.write_file": makeBuiltin(
-		jsonProps(),
+		tree.FunctionProperties{Category: builtinconstants.CategorySystemInfo},
```

Comment:
while you're here could you address https://github.com/cockroachdb/cockroach/issues/84756 by marking `sql_liveness_is_alive` with hidden=true?